CONTRIBUTING.md

```diff
@@ -49,7 +49,7 @@ To report a security issue, please **DO NOT** start by filing a public issue or
 We encourage your participation asking questions and helping improve the FoundationDB project. Check out the [FoundationDB community forums](https://forums.foundationdb.org), which serve a similar function as mailing lists in many open source projects. The forums are organized into three sections:
```

Comment:
Maybe change "sections" here to "categories" as well.

Reply:
Thanks, I didn't see that.